packages/flutter_tools/lib/src/reporting/usage.dart

```diff
@@ -10,7 +10,296 @@ const String _kFlutterUA = 'UA-67589403-6';
 /// When adding to this list, first ensure that the custom dimension is
 /// defined in the backend, or will be defined shortly after the relevant PR
 /// lands.
-enum CustomDimensions {
+class CustomDimensions {
```

Comment:
So that we can find the custom dimensions, I think these fields should contain a comment either linking them to the enum or the cd value

Reply:
Added the comment with cd values. Since the cd value should never change, it should be safe to have the cd value in two separate comments.